install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -13,6 +13,7 @@ data:
       "clusterAPIControllerBareMetal": "quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0",
       "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0",
       "clusterAPIControllerGCP": "quay.io/openshift/origin-gcp-machine-controllers:v4.0.0",
+      "clusterAPIControllerOvirt": "quay.io/openshift/origin-ovirt-machine-controllers",
```

Comment:
Is it okay to omit the tag here?

Reply:
this is fine, this is purely an arbitrary place holder. We have a ticket for next sprint to consolidate on dummy placeholders to remove confusion, e.g  https://github.com/openshift/machine-config-operator/pull/750/files

Reply:
Sounds good.

Reply:
its probably my bad, will add it. can you confirm v4.0.0 is the right one?